app/controllers/mixins/containers_common_logging_mixin.rb

```diff
@@ -0,0 +1,17 @@
+module ContainersCommonLoggingMixin
+  def launch_common_logging
+    record = self.class.model.find(params[:id])
+    ems = record.ext_management_system
+    route_name = ems.common_logging_route_name
```

Comment:
should handle `nil` result
https://github.com/ManageIQ/manageiq/pull/13319#discussion_r97545684